pkg/private/deb/deb.bzl

```diff
@@ -34,79 +34,82 @@ def _pkg_deb_impl(ctx):
         package_file_name = package_file_name,
     )
 
-    changes_file = ctx.actions.declare_file(output_name.rsplit(".", 1)[0] + ".changes")
+    out_file_name_base = output_name.rsplit(".", 1)[0]
```

Comment:
Have you considered using skylib's `paths` module to handle path-related manipulations and extraction?